modules/swagger-codegen/src/main/resources/php/configuration.mustache

```diff
@@ -513,11 +513,13 @@ class Configuration
      */
     public static function toDebugReport()
     {
-        $report  = "PHP SDK ({{invokerPackage}}) Debug Report:\n";
+        $report  = 'PHP SDK ({{invokerPackage}}) Debug Report:'."\n";
```

Comment:
What about replacing "\n" with PHP_EOL? (I thought you've done it via another PR already, maybe you need to rebase on the latest master)

What about remaining lines with double quote? shall we replace it with single quote as well?